### PR TITLE
Implemented putUserInfo, refactored, fixed bugs (wopitest)

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
Implemented WOPI PutUserInfo (https://wopi.readthedocs.io/projects/wopirest/en/latest/files/PutUserInfo.html)
Refactored. Fixed bugs found using wopitest. 
WOPI Validation current status: 70 skipped/65 passed/0 failed

### CODE REVIEW

File: services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java [hunk 1]
```diff
@@ -484,39 +488,46 @@ public Response files(@Context UriInfo uriInfo,
...
-      }
       String lock = request.getHeader(LOCK);
       return delete(config, lock);
     }
+    case PUT_USER_INFO: {
+      try {
+        String userInfo = convertStreamToString(request.getInputStream());
+        return putUserInfo(config, userInfo);
+      } catch (IOException e) {
+        return Response.status(Status.INTERNAL_SERVER_ERROR)
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java:513]: Seems we need LOG.error with related information here.

File: services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java [hunk 2]
```diff
@@ -537,7 +548,7 @@ public Response checkFileInfo(@Context UriInfo uriInfo, @Context HttpServletRequ
       verifyProofKey(request);
     } catch (ProofKeyValidationException e) {
       if (LOG.isDebugEnabled()) {
-        LOG.debug("Proof key validation failed for checkFileInfo", e);
+        LOG.debug("Proof key validation failed for checkFileInfo");
       }
       return Response.status(Status.INTERNAL_SERVER_ERROR)
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java:554]: If it's server error, we need log it as error or warn message and details to be able answer user problem reports.

File: webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml [hunk 3]
```diff
@@ -294,11 +314,11 @@
   </external-component-plugins>
   
   
-   <!-- ECMS menu action for onlyoffice. -->
+   <!-- ECMS menu action for office online. -->
   <external-component-plugins>
     <target-component>org.exoplatform.webui.ext.UIExtensionManager</target-component>
     <component-plugin>
-      <name>Add Onlyoffice Actions</name>
+      <name>Add Officeonline Actions</name>
```
Comment (pnedonosko) [inline webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml]: Office Online

**Review by pnedonosko — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml (hunk 3 above)
Comment (VladMartyshevskyi) [reply webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml]: Fixed: https://github.com/exo-addons/office-online/pull/14/commits/aa44d12709f06db1b8a34dd963511b92d2ecada0

File: services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java:513]: Fixed: https://github.com/exo-addons/office-online/pull/14/commits/ca31f5a28a2aca8b055bb63bea9c00c314d8870a

File: services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java (hunk 2 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java:554]: It's client error that means the request came not from WOPI client (missing proof key headers). But in that case, WOPI requires 500 status code for the response.

**Review by pnedonosko — APPROVED** [review]

**Pull request merged**